lib/rpmds.cc

```diff
@@ -939,12 +871,6 @@ int rpmdsCompareIndex(rpmds A, int aix, rpmds B, int bix)
 	goto exit;
     }
 
-    /* XXX rpm prior to 3.0.2 did not always supply EVR and Flags. */
```

Comment:
Just checking - if this supposed to be dropped? There are some other RPM 3.x conditionals still in place (in other functions in this file).

Reply:
Yep, again related to b3b87f803ccb095b68ac660df9df43dcf0290b4e but this little chicken separated from the others. Should remove the rest of those comments too, I guess the question was where exactly because the issue is/got muddled up between these PRs.